grpc-server/workflow.go

```diff
@@ -267,35 +268,29 @@ func createYaml(ctx context.Context, sqlDB *sql.DB, temp string, tar string) (st
 	if err != nil {
 		return "", err
 	}
-	tarData, err := db.TargetsByID(ctx, sqlDB, tar)
-	if err != nil {
-		return "", err
-	}
-	return renderTemplate(string(tempData), []byte(tarData))
+	return renderTemplate(string(tempData), []byte(tar))
 }
 
 func renderTemplate(tempData string, tarData []byte) (string, error) {
```

Comment:
Please rename `tarData` to something more meaningful.

Reply:
replaced it with `devices`